cmd/cmd.go

```diff
@@ -1162,7 +1170,8 @@ func NewCLI() *cobra.Command {
 
 	runCmd.Flags().String("keepalive", "", "Duration to keep a model loaded (e.g. 5m)")
 	runCmd.Flags().Bool("verbose", false, "Show timings for response")
-	runCmd.Flags().Bool("insecure", false, "Use an insecure registry")
+	runCmd.Flags().Bool("insecure", false, "Do not verify registry TLS")
+	runCmd.Flags().Bool("http", false, "Use an http registry instead of https")
```

Comment:
What happens if I set this _and_ `-insecure`? Is it a nop, an error? It's confusing to have these two flags. It's always confusing to have flags to get canceled out, silently, by another.

I would also be unfortunate to have the two flags, when they could be one, and without breaking the existing behavior that `-insecure` has today.

A (very serious) proposal:

Change the flag to be  _set_ (or simply present), like: `-insecure[="https-only" | "http-only" | "skip-verify"]`. The rules are:

- If present but without a value (e.g. `-insecure`) it defaults to http-only, which is the behavior today.
- If present and set, do what the value says
- If not present, default to `https-only`

The values mean:

- `https-only`: Use HTTPS, always verify TLS
- `skip-verify`: Use HTTPS, but skip TLS verification
- `http-only`: Use HTTP, only. There is no TLS to skip or verify.

> NOTE: The https-only value is there for completeness and should be allowed as a value even though it could be "set" but not using the flag at all.

This is not the most ideal situation to be in, I know. However, the flag was unfortunately introduced to mean something other that "skip TLS verification", so here we are.

In conclusion, the above proposal allows us to maintain the current behavior/meaning of the `-insecure` flag, and allows us to avoid introducing an admittedly ambiguous and confusing `-http` flag.